### ISSUE TITLE
[doc] gcr.io/k8s-staging-cluster-api/capd-manager-amd64:dev not found

### ISSUE THREAD

**Issue by lsytj0413** [kubernetes-sigs/cluster-api#3200]
**What steps did you take and what happened:**
[A clear and concise description on how to REPRODUCE the bug.]

In the [clusterctl for Developers](https://cluster-api.sigs.k8s.io/clusterctl/developers.html#additional-steps-in-order-to-use-the-docker-provider), it use gcr.io/k8s-staging-cluster-api/capd-manager-amd64:dev for the docker provider image, but if we pull it, the docker report **not found**:

>  docker pull gcr.io/k8s-staging-cluster-api/capd-manager-amd64:dev
> Error response from daemon: manifest for gcr.io/k8s-staging-cluster-api/capd-manager-amd64:dev not found: manifest unknown: Failed to fetch "dev" from request "/v2/k8s-staging-cluster-api/capd-manager-amd64/manifests/dev".

**What did you expect to happen:**

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Cluster-api version:
- Minikube/KIND version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

/kind bug
[One or more /area label. See https://github.com/kubernetes-sigs/cluster-api/labels?q=area for the list of labels]

**Comment by lsytj0413** [comment]
Found it must be build from **make -C test/infrastructure/docker docker-build REGISTRY=gcr.io/k8s-staging-cluster-api** command. close issue.

**Comment by fabriziopandini** [comment]
Great to hear you managed to find the solution to your problem.
If you have suggestions on how to make this more clear in the docs, let us know

**Comment by fabriziopandini** [comment]
@Emc1992
https://cluster-api.sigs.k8s.io/clusterctl/developers.html#additional-steps-for-the-docker-provider might help you
TL;DR; build images, then build the manifests, run the local overrides script, then kind load the image

FYI: https://github.com/kubernetes-sigs/cluster-api/pull/3514 is going to simplify the CAPI quick start for CAPD starting from the v0.3.9 release (~ end of the month)

**Comment by Emc1992** [comment]
Yea I needed to change some namespaces correctly to get it working, once I did the above it was looking at the capd-system ns so a service but it was created in the system ns. Once I get some designated time to looking at this, I will I should be able to assist
